lib/check-minor-version.js

```diff
@@ -0,0 +1,16 @@
+var semver = require('semver')
+
+module.exports = function () {
+  require('electron-version')(function (err, electronVersion) {
```

Comment:
Looks like `electron-version` just spawns `electron`: https://github.com/ralphtheninja/electron-version/blob/master/index.js#L8

Hey @ralphtheninja would you be keen on a change to `electron-version` that allows it to take a custom electron path?